db/seeds.rb

```diff
@@ -25,8 +25,13 @@ def print_seed_info(text, indent: 0, colour: nil, blank_lines_before: 0)
   priority_seeds.index(filename.chomp(".rb")) || Float::INFINITY
 end
 
-seed_files.each do |seed_file|
-  puts "\r\n🪴 planting #{seed_file}"
+DeclarativeUpdates.skip do
+  seed_files.each do |seed_file|
+    puts "\r\n🪴 planting #{seed_file}"
 
-  ApplicationRecord.transaction { load(seed_file) }
+    ApplicationRecord.transaction { load(seed_file) }
+  end
 end
+
+# Refresh all metadata
+Metadata::Manager.refresh_all_metadata!(async: true, track_changes: false)
```

Comment:
Out of interest; what was it in the metadata that was causing a hang? was it just volume and if you wait long enough it completes or did it hit some kind of death loop we should investigate?

My only concern is having it `async: true` will trip people up; I know I don't run jobs in development usually. It looks like it takes a while for me with `async: false` though when I tested; maybe we stick with `async: true` for now and switch it back as part of the handler optimisations?